content/docs/for-platform-operators/tutorials/lifecycle/_index.md

```diff
@@ -5,4 +5,434 @@ expand=true
+       declare -x SHLVL="1"
+     layers_dir: /tmp/example/layers/samples_hello-world
+     plan_path: /tmp/samples_hello-world-576309032/samples_hello-world/plan.toml
+     plan contents:
+       [[entries]]
+         name = "some-world"
+       
+       [[entries]]
+         name = "some-world"
+         [entries.metadata]
+           world = "Earth-616"
+---> Done
+Processing layers
+Updating environment
+Reading output files
+Updating buildpack processes
+Updating process list
+Finished running build for buildpack samples/hello-world@0.0.1
+Running build for buildpack samples/hello-moon@0.0.1
+Looking up buildpack
+Finding plan
+Creating plan directory
+Preparing paths
+Running build command
+---> Hello Moon buildpack
+     env_dir: /tmp/example/platform/env
+     env vars:
+       declare -x CNB_BP_PLAN_PATH="/tmp/samples_hello-moon-3356528850/samples_hello-moon/plan.toml"
+       declare -x CNB_BUILDPACK_DIR="/tmp/example/buildpacks/samples_hello-moon/0.0.1"
+       declare -x CNB_LAYERS_DIR="/tmp/example/layers/samples_hello-moon"
+       declare -x CNB_PLATFORM_DIR="/tmp/example/platform"
+       declare -x CNB_TARGET_ARCH="amd64"
+       declare -x CNB_TARGET_DISTRO_NAME="ubuntu"
+       declare -x CNB_TARGET_DISTRO_VERSION="22.04"
+       declare -x CNB_TARGET_OS="linux"
+       declare -x HOME="/home/gitpod"
+       declare -x HOSTNAME="buildpacks-docs-dusxugo5ehi"
+       declare -x OLDPWD
+       declare -x PATH="/bin:/usr/bin:/usr/local/bin"
+       declare -x PWD="/tmp/example/workspace"
+       declare -x SHLVL="1"
+     layers_dir: /tmp/example/layers/samples_hello-moon
+     plan_path: /tmp/samples_hello-moon-3356528850/samples_hello-moon/plan.toml
+     plan contents:
+---> Done
+Processing layers
+Updating environment
+Reading output files
+Updating buildpack processes
+Updating process list
+Finished running build for buildpack samples/hello-moon@0.0.1
+Copying SBOM files
+Creating SBOM files for legacy BOM
+Listing processes
+Timer: Builder ran for 20.200892ms and ended at 2024-10-01T07:07:46Z
+```
+
+#### Export
+
+The purpose of the `export` phase is to create a new `OCI` image using a combination of remote layers, local `<layers>/<layer>` layers, and the processed `app` directory.
+
+To export the artifacts built by the `builder`, you first need to specify the path of the `launcher` that your image is going to run:
+
+* For AMD64 architectures
+  
+  ```text
+  export CNB_LINUX_LAUNCHER_PATH=/<your-path>/lifecycle/out/linux-amd64/lifecycle/launcher
+  ```
+
+* For ARM64 Architectures
+
+  ```text
+  export CNB_LINUX_LAUNCHER_PATH=/<your-path>/lifecycle/out/linux-arm64/lifecycle/launcher
+  ```
+
+Now you can run the `exporter`:
+
+```text
+${CNB_LIFECYCLE_PATH}/exporter --log-level debug -launch-cache "./cache" -daemon -cache-dir "./cache" -analyzed "./layers/analyzed.toml" -group "./layers/group.toml" -layers="./layers" -app "./workspace" -launcher="${CNB_LINUX_LAUNCHER_PATH}" -process-type="shell" apps/bash-script
```

Comment:
```suggestion
${CNB_LIFECYCLE_PATH}/exporter --log-level debug -launch-cache "./cache" -daemon -cache-dir "./cache" -analyzed "./layers/analyzed.toml" -group "./layers/group.toml" -layers="./layers" -app "./workspace" -launcher="${CNB_LINUX_LAUNCHER_PATH}" apps/bash-script
```

If we don't hand-create that `./layers/samples_hello-moon/launch.toml` file as I suggest above, we should remove the `-process-type` flag

Reply:
I also see 

```
Warning: Failed to export cache: setting cache metadata: creating metadata file '/Users/na001548/workspace/scratch/build-root/cache/staging/io.buildpacks.lifecycle.cache.metadata': open /Users/na001548/workspace/scratch/build-root/cache/staging/io.buildpacks.lifecycle.cache.metadata: no such file or director
```

And, IDK why (we should investigate cc @AidanDelaney) but the build doesn't fail which is as intended

Reply:
I updated the command; please let me know if this is correct.